Features/ContentTypeManagement/field_definitions_automatic_ordering.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Automatic ordering of field definitions when editing a content type
+  In order to manage content types
+  As an Administrator user
+  I need to be able to see field definitions sorted in a sensible, predictable order
+
+  Background:
+    Given I am logged in as admin on PlatformUI
+
+  ##
+  #
+  # Order by position
+  #
+  ##
+  @javascript @common
+  Scenario: Order field types of Article content type by position
+    Given an "Article" content type exists
+    And I am on the "Article" content type view page
+    When I edit the "Article" content type
+    Then I should see the following field definitions:
```

Comment:
While this sentence is generic, I'm not sure that it really "covers" what we want to verify.

This scenario alone doesn't make that much sense: is it really an expectation that you see the fields in a particular order ? It feels more like an assertion of the system's state, which should probably not be done as a scenario.

Maybe i'd remove this particular scenario, as it doesn't have any real action.